### DISCUSSION TITLE
What's implied with 1 user when using private packages?

### DISCUSSION THREAD

**Discussion by tomosterlund** [community/community#123608] [npm] [answered]
At the pricing page, I can't find any definition of what a "user" is. It says the price per user is 7$, but my question would be:

- Is a user **a person with an account at npm, that can collaborate within my organization**?
- Is a user **any person that interacts with my private packages**?

If the former is the case, this makes npm private packages the hottest candidate for me to distribute closed source code to my users.

If the latter is the case, that would be way too expensive for me, since I would potentially pay $7 each, for dozens of people who just download my assets.

**Comment by Alireza-Valizadeh** [comment, marked as answer]
Both of them are correct. 
[_If you have an organization with a private packages plan, your organization will cost you seven (7) dollars a month per user. The $7 charge is a flat fee for any member of the organization even if the teams the member belongs do not have access to private packages_](https://docs.npmjs.com/upgrading-to-a-paid-organization-plan)
The monthly amount will be the number of members in your organization multiplied by $7. But then if someone wants to use that package they either have to be a member of your organization to have access to that package or you can directly grant read (or read-write) permissions to your user's NPM account for that specific package, just like private git repositories.
